homeassistant/components/fritzbox/const.py

```diff
@@ -30,4 +30,5 @@
     Platform.LIGHT,
     Platform.SWITCH,
     Platform.SENSOR,
+    Platform.COVER,
```

Comment:
please sort this list alphabetically (_please be so kind to also correct sorting of sensor and switch, thx_)